assemblyline/odm/models/ontology/results/network.py

```diff
@@ -46,6 +46,12 @@ class NetworkHTTP(odm.Model):
         odm.Text(), description="The response content mimetype returned by the server")
 
 
+@odm.model(description="Details for an SMTP request")
+class NetworkSMTP(odm.Model):
+    mail_from = odm.Email(description="Sender of the email")
+    mail_to = odm.List(odm.Email(), description="Recipients of the email")
+    attachments = odm.List(odm.Text(), default=[], description="Name of attachments, if any")
```

Comment:
Should this be `attachment_names`, I could see an attachments entry being an `odm.List(odm.Compound(File))`, which also includes a list of names. Should we go straight to a File object? I'm pretty sure we could fill it in in EmlParser already. Do you have that file extracted in Suricata so that you could call fileinfo on it?

Reply:
🤔🤔🤔